tests/plugins/sqlite/SqliteEventViewTest.cpp

```diff
@@ -123,6 +127,102 @@ void SqliteEventViewTest::testFilterWithValueToExclude()
     delete view;
 }
 
+void SqliteEventViewTest::testFilterWithFilterLessThan()
+{
+    History::Sort ascendingSort(History::FieldEventId, Qt::AscendingOrder);
+    History::IntersectionFilter filter;
+
+    QDateTime queryTime = QDateTime::currentDateTime().addDays(10);
+    filter.append(History::Filter(History::FieldAccountId, "account0"));
+    filter.append(History::Filter(History::FieldTimestamp, queryTime, History::MatchLess));
+    History::PluginEventView *view = mPlugin->queryEvents(History::EventTypeText, ascendingSort, filter);
+    QVERIFY(view->IsValid());
+    QList<QVariantMap> events = view->NextPage();
+    while (!events.isEmpty()) {
```

Comment:
I don't like potentially endless loops in test functions if they can be avoided. Maybe there's another way to get the list of events that doesn't require a while loop?

Reply:
Well i don't see alternative for that. seems to be used in e.g testNextPage()